features/admin.actions.v1/components/pre-update-password-action-config-form.tsx

```diff
@@ -109,6 +116,14 @@ const PreUpdatePasswordActionConfigForm: FunctionComponent<PreUpdatePasswordActi
         mutate: mutateAction
     } = useGetActionById(actionTypeApiPath, initialValues?.id);
 
+    const {
+        data: RuleExpressionsMetaData
+    } = useGetRulesMeta(actionTypeApiPath);
+
+    // TODO: Temporary flag to show/hide the rule component.
+    const showRuleComponent: boolean = isFeatureEnabled(
+        actionsFeatureConfig, "actions.types.preUpdatePassword.edit.rule");
```

Comment:
Get the feature id from a constant. We usually maintain these under a FEATURE DICTIONARY map.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7486/commits/34a926448e3cea96642355bfe473951501c89739